prime-dotnet-webapi/Controllers/OrganizationsController.cs

```diff
@@ -308,19 +308,25 @@ public async Task<ActionResult<Agreement>> GetOrganizationAgreement(int organiza
         /// Get the organization agreement as a signable PDF, Base 64 encoded.
         /// </summary>
         /// <param name="organizationId"></param>
-        /// <param name="agreementId"></param>
-        [HttpGet("{organizationId}/agreements/{agreementId}/signable", Name = nameof(GetSignableOrganizationAgreement))]
+        /// <param name="agreementType"></param>
+        [HttpGet("{organizationId}/signable", Name = nameof(GetSignableOrganizationAgreement))]
         [ProducesResponseType(typeof(ApiBadRequestResponse), StatusCodes.Status400BadRequest)]
         [ProducesResponseType(StatusCodes.Status401Unauthorized)]
         [ProducesResponseType(StatusCodes.Status403Forbidden)]
         [ProducesResponseType(typeof(ApiMessageResponse), StatusCodes.Status404NotFound)]
         [ProducesResponseType(typeof(ApiResultResponse<string>), StatusCodes.Status200OK)]
-        public async Task<ActionResult<string>> GetSignableOrganizationAgreement(int organizationId, int agreementId)
+        public async Task<ActionResult<string>> GetSignableOrganizationAgreement(int organizationId, [FromQuery] AgreementType agreementType)
         {
-            var pdf = await _agreementService.GetSignableOrgAgreementAsync(organizationId, agreementId);
+            var allowedAgreementTypes = new[] { AgreementType.CommunityPracticeOrgAgreement, AgreementType.CommunityPharmacyOrgAgreement };
```

Comment:
We have an Extension method for IsEnrolleeAgreement. I would prefer to see this as either !IsEnrolleeAgreement or as a new Extension method IsOrganizationAgreement